### PR TITLE
feat: new label dropdown using dropdown component

### PR DESCRIPTION
**Pull request by iamsivin**
https://user-images.githubusercontent.com/64252451/117924114-df2d4000-b312-11eb-9f23-ee74412c57d4.mov

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/conversation/labels/LabelBox.vue [hunk 1]
```diff
@@ -100,13 +113,24 @@ export default {
...
-      bus.$emit('fetch_conversation_stats');
-      this.isEditing = false;
+
+    addItem(value) {
+      this.selectedLabels.push(value.title);
+      this.onUpdateLabels(this.selectedLabels);
+    },
+
+    removeItem(value) {
+      this.selectedLabels.filter(label => label.title === value.title);
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/conversation/labels/LabelBox.vue:127]: `this.selectedLabels.filter(label => label.title === value.title);` will return a new array without `value.title`. Assign the result this to a new `const` and assign it to  `this.selectedLabels`. Then call `this.onUpdateLabels(this.selectedLabels);`

**Review by nithindavid — CHANGES_REQUESTED** [review]

File: app/javascript/shared/components/ui/dropdown/AddLabel.vue [hunk 2]
```diff
@@ -22,4 +21,14 @@ export default {
 };
 </script>
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+.button-wrap {
+  padding: var(--space-micro) var(--space-small);
```
Comment (iamsivin) [inline app/javascript/shared/components/ui/dropdown/AddLabel.vue:26]: To be removed button component tiny button styles are fixed.

Comment (iamsivin) [inline app/javascript/shared/components/ui/dropdown/AddLabel.vue:26]: To be removed when button component tiny button styles are fixed.

**Comment by nithindavid** [comment]
@iamsivin Please move the conversations block to the end of list as Pranav mentioned.

**Review by nithindavid — CHANGES_REQUESTED** [review]
👆

File: app/javascript/shared/components/ui/dropdown/AddLabel.vue [hunk 3]
```diff
@@ -22,4 +21,14 @@ export default {
...
+  display: inline;
+  line-height: 1.4rem;
```
Comment (nithindavid) [inline app/javascript/shared/components/ui/dropdown/AddLabel.vue:28]: Use variable here.

**Review by nithindavid — CHANGES_REQUESTED** [review]

**Review by nithindavid — APPROVED** [review]

**Comment by pranavrajs** [comment]
@nithindavid I have tested this out on staging. Can we merge it? Is there something left to be done on this PR?

**Pull request merged**